xml/System.Diagnostics.Tracing/IncrementingEventCounter.xml

```diff
@@ -34,9 +34,9 @@
         <Parameter Name="eventSource" Type="System.Diagnostics.Tracing.EventSource" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <param name="eventSource">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The name.</param>
+        <param name="eventSource">The event source.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Diagnostics.Tracing.IncrementingEventCounter" /> class. IncrementingEventCounter live as long as the EventSource that they are attached to unless they are explicitly disposed.</summary>
```

Comment:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Diagnostics.Tracing.IncrementingEventCounter" /> class. IncrementingEventCounters live as long as the EventSource that they are attached to unless they are explicitly disposed.</summary>
```

Reply:
Thank you @noahfalk! Yes, this is totally the right place to make any adjustments to the descriptions.

The dotnet-api-docs repo is considered the source of truth for corefx/coreclr API documentation. The triple slash comments are simply being used as the seed.

Reply:
I made an additional modification to your suggestion.
Also, since we are pluralizing, I'm wondering if it would be better to describe the object with simple language instead. @mairaw @rpetrusha what's the rule for pluralizing types?

```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Diagnostics.Tracing.IncrementingEventCounter" /> class. Incrementing event counters live as long as the <see cref="T:System.Diagnostics.Tracing.EventSource" /> that they are attached to unless they are explicitly disposed.</summary>
```